src/ui/events.js

```diff
@@ -569,13 +622,18 @@ export type MapEvent =
      * @instance
      * @property {MapWheelEvent} data
      * @example
-     * // Initialize the map
-     * var map = new mapboxgl.Map({ // map options });
-     * // Set an event listener that fires
-     * // when a wheel event occurs within the map.
-     * map.on('wheel', function() {
-     *   console.log('A wheel event occurred.');
-     * });
+     * // Example of a MapDataEvent of type "sourcedata"
+     * {
+     *   type: "sourcedata",
+     *   dataType: "source",
+     *   isSourceLoaded, false,
+     *   source: {...},
+     *   style: {...}
+     *   sourceDataType: "",
+     *   sourceId: "composite",
+     *   tile: {...},
+     *   coord: {...}
+     * }
```

Comment:
```suggestion
     * // Example of a MapDataEvent of type "sourcedata"
     * {
     *   type: "sourcedata",
     *   dataType: "source",
     *   isSourceLoaded, false,
     *   source: {...},
     *   style: {...}
     *   sourceDataType: "",
     *   sourceId: "composite",
     *   tile: {...},
     *   coord: {...}
     * }
```
remove misplaced example